INSTALLATION.md

```diff
@@ -26,6 +26,7 @@ sudo apt-get install python3 python3-pip
 ```
 
 ```shell
+pip3 install license_expression
```

Comment:
That is not needed at all - `pip` automatically install all the mentioned dependencies from `requirements.txt`.
All we need is a release to pypi.org that actually contains this file (so just bump a 0.1.1 or 0.2 version and we're good)

Reply:
@hesa can we close it with that statement?